.gitlab-ci.yml

```diff
@@ -180,4 +180,4 @@ deploy:sync-cms:
   rules:
     - if: $CI_COMMIT_BRANCH == 'master'
       changes:
-        - dev/squidex
+        - dev/squidex/**/*
```

Comment:
Should we add dev/squidex/*` also? Or does the glob matches files on root?

Reply:
`      - dockerfiles/**/*`
> Any of the files and subdirectories inside the dockerfiles directory.

Reply:
I think it will track everything. 😉